ads/_config.js

```diff
@@ -1079,6 +1079,10 @@ export const adConfig = {
     preconnect: 'https://yads.yahoo.co.jp',
   },
 
+  'yahoonativeads': {
+    renderStartImplemented: true,
```

Comment:
I don't see this callback triggered. Sorry, maybe I wasn't clear, you will actually need to call this function in either your `readmo.js` or in your `yahoonativeads.js` (less common). If you have this flag as true without calling it, it will actually take longer to render. You can either call this in your js or remove this config option. there is a bit more detail [here](https://github.com/ampproject/amphtml/blob/master/ads/README.md#available-apis)

Reply:
I decided to use 
```
loadScript(global, 'https://s.yimg.com/dy/ads/readmo.js', () => global.context.renderStart());
```
will that suffice?

Reply:
works for me :) you may want to move this to your remote js at some point, but this is fine.

Reply:
prettier wants this on a separate line though. 
```js
loadScript(global, 'https://s.yimg.com/dy/ads/readmo.js', () =>
  global.context.renderStart()
);
```

Reply:
✅ Done